mage_ai/frontend/components/RuntimeVariables/index.style.tsx

```diff
@@ -1,28 +1,12 @@
 import styled from 'styled-components';
 
 import dark from '@oracle/styles/themes/dark';
-import { BORDER_RADIUS } from '@oracle/styles/units/borders';
-import { PADDING, UNIT } from '@oracle/styles/units/spacing';
-import { ScrollbarStyledCss } from '@oracle/styles/scrollbars';
-
-export const ContainerStyle = styled.div`
-  border-bottom: 1px solid ${dark.borders.medium};
-  padding: ${PADDING}px;
-`;
-
-export const CardsStyle = styled.div`
-  ${ScrollbarStyledCss}
-
-  height: 80px;
+import { PADDING } from '@oracle/styles/units/spacing';
 
+export const ContainerStyle = styled.div<{ $height: number }>`
   display: flex;
-  overflow-x: scroll;
-`;
-
-export const VariableCardStyle = styled.div`
-  background-color: ${dark.background.output};
-  border-radius: ${BORDER_RADIUS}px;
-  flex-shrink: 0;
-  margin-right: ${UNIT}px;
+  flex-direction: column;
+  height: ${props => props.$height}px;
```

Comment:
We set a fixed height here so that overflow for the table will be triggered with > 5 variables.